Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -140,52 +135,44 @@ public class ProjectController : Controller
       * @apiSuccessExample {json} Success-Response:
       * HTTP/1.1 200 OK 
       * {
-      *     "description":"Add Project in ProjectTable"
+      *     "description":"Add new project"
       * }
       */
         [Authorize]
         [HttpPost]
-        [Route("addProject")]
+        [Route("")]
         public async Task<IActionResult> addProject([FromBody]ProjectAc project)
         {
-            try
+            var createdBy = _userManager.GetUserId(User);
+            if (ModelState.IsValid)
             {
-                var createdBy = _userManager.GetUserId(User);
-                if (ModelState.IsValid)
+                ProjectAc projectAc = _projectRepository.checkDuplicate(project);
+                if (projectAc.Name != null && projectAc.SlackChannelName != null)
```

Comment:
I have update my code now I have use string.isnullorempty.
Firstly I have check project and slack channel name already exists or not if project and slack channel name exits then I have return null (to easily identify and also notify end user which are already exists.). that's why i have again check project and slack channel name is null or not.